src/routes/(settings)/settings/billing/canceled/unlimited/+page.svelte

```
@@ -0,0 +1,76 @@
+<script lang="ts">
+  import { Button } from "$lib/components/ui/button";
+  import { XCircle, ArrowLeft, Infinity } from "@lucide/svelte";
```

ESLint

[error] Shadowing of global property 'Infinity'.

(no-shadow-restricted-names)